Aquality.Appium.Mobile/src/Aquality.Appium.Mobile/Screens/IScreen.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Aquality.Selenium.Core.Elements.Interfaces;
+using OpenQA.Selenium;
+using System.Drawing;
+
+namespace Aquality.Appium.Mobile.Screens
+{
+    /// <summary>
+    /// Interface that describes application under the test screen.
+    /// </summary>
+    public interface IScreen 
+    {
+        /// <summary>
+        /// Unique screen locator.
+        /// </summary>
+        By Locator { get; }
+
+        /// <summary>
+        /// Unique screen name.
+        /// </summary>
+        string Name { get; }
+
+        /// <summary>
+        /// Gets Displayed screen's state by its locator.
+        /// </summary>
+        bool IsDisplayed { get; }
```

Comment:
let's remove it, current implementation 
`public bool IsDisplayed => State.WaitForDisplayed();`
Is a bit confusing

Reply:
In this case it makes sense to remove public IsDisplayed from Screen too.